src/FacetedTokenInput.js

```diff
@@ -149,10 +197,105 @@ export default class FacetedTokenInput extends Component {
         { this.props.children }
 
         { showDropDown && this.renderDropdown() }
+
+
+
+        { /* start test code */ }
+
+        { /*
+          the following div needs to be hidden if this code is going into the
+          master branch (see A11Y_HELPER_STYLE)
+          style={ A11Y_HELPER_STYLE }
+          */ }
+        <div
+          id="a11ysupport"
+          role="log"
+          aria-live="polite"
+          aria-relevant="all"
+          aria-atomic={ true }
+          style={ A11Y_HELPER_STYLE }
+        >
+
+        { /*
+          <span id="tagsupport">
+              simply add all the tokens that exist in the input
+
+              <p></p>
+
+              this adds only the last token, it doesn't solve the current
+              problem since the additions count from the second one onwards
+
+              (tokens.length) ? this.a11ySupport(tokens[tokens.length - 1], tokens.length - 1) : ''
+
+              GOOGLE
+              <div aria-live="polite" aria-atomic="true">
+                pierpaolo.ramon aggiunto. Premi backspace per rimuovere il destinatario.
+              </div>
+
+
+              { tokens.map(this.a11ySupport, this) }
+          </span>
+        */ }
+
+          { this.a11ySupport() }
+        </div>
```

Comment:
this container is a live region and should read addition and removals of tokens. 
`aria-atomic` is set to true because the entire phrase needs to be read and not only whats get changed